README.md

```diff
@@ -93,20 +93,32 @@ Multi-noding (Sharing CPU, RAM, Uplink or storage among nodes) is not allowed.**
 
    You can use the text editor of your choice (e.g. `nano` or `vim` on Linux)
 
-4. Download the docker-compose file
+4. Download the `docker-compose.yml` file:
 
    ```bash
    curl -s https://raw.githubusercontent.com/filecoin-saturn/L1-node/main/docker-compose.yml -o docker-compose.yml
    ```
 
-5. Launch it:
+5. Set up the cron job to auto-update the `docker-compose.yml` file:
+
+   ```bash
+   crontab -e
+   ```
+
+   Add the following text to your editor, making sure to replace `$SATURN_HOME` with its value:
+
+   ```
+   */5 * * * * cd $SATURN_HOME && curl -s https://raw.githubusercontent.com/filecoin-saturn/L1-node/main/docker-compose.yml -o docker-compose.yml.curl && diff docker-compose.yml.curl docker-compose.yml >/dev/null || (mv -f docker-compose.yml.curl docker-compose.yml && sudo docker compose up -d)
```

Comment:
```suggestion
   */5 * * * * cd $SATURN_HOME && curl -fs https://raw.githubusercontent.com/filecoin-saturn/L1-node/main/docker-compose.yml -o docker-compose.yml.curl && diff docker-compose.yml.curl docker-compose.yml >/dev/null || (mv -f docker-compose.yml.curl docker-compose.yml && sudo docker compose up -d)
```

Fail on HTTP errors, in order not to swallow issues

Reply:
If we move this logic to a script, could we also auto-update this logic? This way we also don't need to mirror the logic in the ansible playbook

Reply:
Keeping this DRY makes sense. 

What about having the script fetch itself? The part I'm worried about is having another script that gets updated transparently and which node operators may not appreciate.

Alternatively, we can simply create the script and have operators fetch it as an additional step with no auto-update of its own. What do you think?